build.gradle

```diff
@@ -1,6 +1,6 @@
 
 plugins {
-    id 'nebula.netflixoss' version '8.8.1'
+    id 'nebula.netflixoss' version '9.1.0'
```

Comment:
You'll need to run the dependency lock command in this PR for this to take effect.

Reply:
We don't lock buildscript classpath dependencies with nebula lock so this isn't necessary. Running `generateLock saveLock` only upgrades jmh and mockito-core.

Will add them